### ISSUE TITLE
top 8 in table heading should probably be a lowercase numeral

### ISSUE THREAD

**Issue by vorpal-buildbot** [PennyDreadfulMTG/Penny-Dreadful-Tools#5768]
Reported on Discord by bakert#2193

**Comment by bakert** [comment]
Otherwise known as "old-style". We use these numerals in parts of the site but the application is not quite consistent and not quite right. I'm not even sure the current state of affairs is better than just not using old style numerals. But Practical Typography and The Elements of Typographic Style both tell me I need them and who am I to argue? Modern numerals do stick out like a sore thumb in all-lowercase'd mtgo usernames for starters.

Right now the rules we have are:

```
p, .subtitle, .news .item {
    font-variant-numeric: oldstyle-nums;
}

table, ul.decklist.li, .news .date {
    font-variant-numeric: normal;
}
```

This is trying to represent "in body text/all-lowercase use lowercase nums, in titles/title text use uppercase nums". But it's only partly correct.

**Comment by bakert** [comment]
Current:

![image](https://user-images.githubusercontent.com/1541/52457069-d08cc880-2b0c-11e9-83cd-7f88000e9ace.png)

Proposed:

![image](https://user-images.githubusercontent.com/1541/52457078-dbdff400-2b0c-11e9-9572-b3c32d988d66.png)

I think it *is* better. I'm not completely sure why the small caps variant of the font isn't doing this in the first place.

**Comment by bakert** [comment]
I've set anywhere using heading-caps (the all-small-caps font used in table headings and links) to use oldstyle-nums. And it might be a bit nutty but I've wrapped every use of MTGO username I can find in .person class so I can apply it there too, even in headings. We'll see how it feels

![image](https://user-images.githubusercontent.com/1541/52457530-15196380-2b0f-11e9-859a-15780bc53187.png)

Not:

![image](https://user-images.githubusercontent.com/1541/52457569-5873d200-2b0f-11e9-9172-b8d0b43937e6.png)


And:

![image](https://user-images.githubusercontent.com/1541/52457536-2498ac80-2b0f-11e9-819b-b044107eec55.png)

Not:

![image](https://user-images.githubusercontent.com/1541/52457556-47c35c00-2b0f-11e9-85f1-02c6567cbc77.png)


Slight awkwardness – because the code for the page title/banner is shared I had to put the person class on all h1 headings. We'll see if this bites us anywhere.

450e0189 (HEAD -> blitz) Wrap all uses of mtgo username in a .person class to lowercase numberals.
67576930 Use oldstyle numbers wherever we are using small caps.
